src/main/resources/templates/db_scripts/schema.json

```diff
@@ -63,28 +63,7 @@
       "populateJsonWithId": true
     },
     {
-      "tableName": "contact_person",
-      "generateId": true,
-      "fromModuleVersion": 1.0,
-      "withMetadata": true,
-      "populateJsonWithId": true
-    },
-    {
-      "tableName": "edi",
-      "generateId": true,
-      "fromModuleVersion": 1.0,
-      "withMetadata": true,
-      "populateJsonWithId": true
-    },
-    {
-      "tableName": "edi_ftp",
-      "generateId": true,
-      "fromModuleVersion": 1.0,
-      "withMetadata": true,
-      "populateJsonWithId": true
-    },
-    {
-      "tableName": "edi_job",
+      "tableName": "contact",
```

Comment:
This file needs a little cleanup, there are few extra tables being created like vendor_category, contact_category that are not being used.